lib/twilio/call.ts

```diff
@@ -1862,7 +1852,7 @@ namespace Call {
     /**
      * The type of message
      */
-    messageType: MessageType;
+    messageType: string;
```

Comment:
Since this is now a string, the customer needs to know what the possible values are. Let's update the docstring to the following. Please verify the docs actually renders the link below properly. Run `npm run docs:ts` and open docs/index.html and see the documentation you just updated.
```
/**
 * The type of message. Currently, only 'user-defined-message' is supported. More message types will be added in the future. See [call resource](https://www.twilio.com/docs/voice/api/call-resource) documentation for more details.
 */
```